### ISSUE TITLE
Why Contributor License Agreement (CLA)?

### ISSUE THREAD

**Issue by KenMacD** [livekit/agents#742]
I noticed that PRs require a CLA. Personally this greatly reduces the chances that I would use this project. A CLA basically says to me that the project will change to closed-source/non-free down the road.

It's your project, so do what you like, but I'm sure I'm not the only one that is turned away by the CLA requirement.

For more reading see: https://drewdevault.com/2023/07/04/Dont-sign-a-CLA-2.html

**Comment by willsmanley** [comment]
I disagree here. Proper legal protections are a good idea for any open source project maintainer, especially corporate ones with a lot to potentially lose if someone commits code from someone else's IP. Most projects with a CLA do not ultimately go closed source.
